index.php

```diff
@@ -1 +1,247 @@
+<?php
+$time_start = microtime( true );
+mb_internal_encoding( "UTF-8" );
+if ( isset( $_POST['data'] ) ) {
+	$numberOfCharacter                                    = strlen( $_POST['data'] );
+	$numberOfWords                                        = numberOfWords( $_POST['data'] );
+	$numberOfSentences                                    = numberOfSentences( $_POST['data'] );
+	$frequencyOfCharacters                                = frequencyOfCharacters( $_POST['data'] );
+	$distributionOfCharactersAsAPercentageOfTotal         = distributionOfCharactersAsAPercentageOfTotal( $_POST['data'], $frequencyOfCharacters );
+	$averageWordLength                                    = averageWordLength( $_POST['data'] );
+	$theAverageNumberOfWordsInASentence                   = theAverageNumberOfWordsInASentence( $numberOfSentences, $numberOfWords );
+	$top10MostUsedWords                                   = top10MostUsedWords( $_POST['data'] );
+	$top10longestWords                                    = top10LongestWords( $_POST['data'] );
+	$top10ShortestWords                                   = top10ShortestWords( $_POST['data'] );
+	$top10LongestSentences                                = top10LongestSentences( $_POST['data'] );
+	$top10shortestSentences                               = top10shortestSentences( $_POST['data'] );
+	$numberOfPalindromeWords                              = numberOfPalindromeWords( $_POST['data'] );
+	$top10LongestPalindromeWords                          = top10LongestPalindromeWords( $_POST['data'] );
+	$isTheWholeTextAPalindrome                            = isTheWholeTextAPalindrome( $_POST['data'] );
+	$timeTheReport                                        = date( 'Y-m-d H-i-s' );
+	$theReversedText                                      = theReversedText( $_POST['data'] );
+	$theReversedTextButTheCharacterOrderInWordsKeptIntact = theReversedTextButTheCharacterOrderInWordsKeptIntact( 'This is the text.' );
+}
 
+function numberOfWords( $string ) {
+	$res = explode( ' ', $string );
+
+	return count( $res );
+}
+
+function numberOfSentences( $string ) {
```

Comment:
Please reformat functions declarations and functions calls to meet PSR standard:
https://www.php-fig.org/psr/psr-12/#44-methods-and-functions
https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments